wiki/intro/faq/de.yml

```
@@ -0,0 +1,34 @@
+title: "FAQ"
+description: "Häufig gestellte Fragen"
+
+q-info: "Was macht diese Mod?"
+a-info: "Diese Mod ermöglicht es dir, in JavaScript erstellte Skripte zu verwenden, um deinen Server zu verwalten, neue Blöcke und Items hinzuzufügen, Rezepte zu ändern, eigene Event-Handler für Quest-Mods hinzuzufügen und vieles mehr!"
```

Would change this to

```suggestion
a-info: "Diese Mod ermöglicht es dir, Skripte in der JavaScript-Sprache zu schreiben, um deinen Server zu verwalten, neue Blöcke und Items hinzuzufügen, Rezepte zu ändern, eigene Event-Handler für Quest-Mods hinzuzufügen und vieles mehr!"
```

I think nobody would ever say "Skripte erstellen" although it's the direct translation.